src/Application.php

```diff
@@ -25,9 +25,9 @@
 
 class Application extends BaseApplication
 {
-    public const VERSION = '@git-version@';
+    final public const VERSION = '@git-version@';
 
-    public function __construct()
+    public function __construct(private AnalyzeCommand $analyzeCommand, private RulesCommand $rulesCommand)
```

Comment:
@OskarStark 

Here is the problem.

because `bin/doctor-rst` is still `(new Application())->run();`

And here `buildContainer` method is not used anymore ($container L42 is unused)

Reply:
Indeed, I need to check, what rector is refactoring here, thanks

Reply:
I ignored `src/Application.php`, lets see if this is working again